articles/laravel-8-email-verification/index.md

```diff
@@ -11,11 +11,11 @@ topics: [Languages]
 excerpt_separator: <!--more-->
 images:
   - url: /engineering-education/laravel-8-email-verification/hero.jpg
-    alt: laravel language localisation
+    alt: laravel email verification
 ---
 Laravel is a PHP-based framework that is simple to understand. It makes it easy to code complex features in an application. One of the features that can be implemented quickly using Laravel is email verification.  
 <!--more-->
-Typically, we need to verify users' emails whenever they sign up on the website. This process ensures that we accept only those users with valid email addresses.  In the past, it was quite difficult to code or include email verification in applications. The introduction of Laravel made life better for developers. In other words, implementing email verification in Laravel only requires a few lines of code.
+Typically, we need to verify emails whenever a user signs up on the website. This process ensures that we only accept those users with valid email addresses. In the past, it was quite difficult to code or include email verification in applications. The introduction of Laravel made life better for developers. In other words, implementing email verification in Laravel only requires a few lines of code.
```

Comment:
I still think the previous statement is(was) much better than this minor change!

Reply:
The change I made is okay.